src/server/auth.ts

```
@@ -57,38 +58,57 @@ const getFirstNameAndLastName = (
   };
 };
 
+function generateRandomString(length: number): string {
+  return randomBytes(length).toString('hex').slice(0, length);
+}
+
 const createUserEventHandler = async (message: { user: User }) => {
   const { user } = message;
 
+  if (!user.email) return;
+
   if (!user.name) return;
```

We should throw errors and log here instead of silently returning